src/components/inputs/AttributesList.vue

```diff
@@ -168,7 +170,7 @@ function updateAttributeValue(event) {
 
 /**
  * Delete attribute and emit event to update parent attributes list.
- * @param event
+ * @param {object} event - Event.
  */
 function deleteAttribute(event) {
```

Comment:
check if `event` can be changed to `{attribute}`

Reply:
done